sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 7.6.0-beta.1 (Unreleased)
 
+### Features Added
+- Add `ServiceBusMessageState` and expose it in `ServiceBusReceivedMessage.getMessageState()`. ([#25217](https://github.com/Azure/azure-sdk-for-java/issues/25217))
```

Comment:
Something to indicate why this would be useful?

```suggestion
- Add `ServiceBusMessageState` property to received messages which indicates whether the message is active, scheduled or deferred. It is exposed it in `ServiceBusReceivedMessage.getMessageState()`. ([#25217](https://github.com/Azure/azure-sdk-for-java/issues/25217))
```